client/app/resources/i18n/doubleAuth/Messages_fr_FR.xml

```diff
@@ -4,5 +4,5 @@
    <translation id="doubleauth_modal_alert_message" qtlid="231102">Attention vous ne disposez plus de code de secours à usage unique.</translation>
    <translation id="doubleauth_modal_alert_message_1" qtlid="231115">Attention vous ne disposez plus que d'un seul code de secours à usage unique.</translation>
    <translation id="doubleauth_modal_alert_message_other" qtlid="231128">Attention vous ne disposez plus que de <strong>{0}</strong> codes de secours à usage unique.</translation>
-   <translation id="doubleauth_modal_alert_message_regenerate" qtlid="231141">Voulez-vous regénérer un nouveau jeu de codes de secours ?</translation>
+   <translation id="doubleauth_modal_alert_message_regenerate" qtlid="231141">Voulez-vous régénérer un nouveau jeu de codes de secours ?</translation>
 </translations>
```

Comment:
https://fr.wiktionary.org/wiki/regénérer

Reply:
http://dictionnaire.reverso.net/francais-definition/reg%C3%A9n%C3%A9rer
http://www.larousse.fr/dictionnaires/rechercher?q=reg%C3%A9n%C3%A9rer&l=francais&culture=
http://www.linternaute.com/encyclopedie/recherche/id-195/?f_libelle=reg%C3%A9n%C3%A9rer
http://www.cnrtl.fr/definition/reg%C3%A9n%C3%A9rer

Reply:
Hi, 
If a translation is changed, it is needed to replicate it on Translator II tools or to remove the qtlid.
About terms question, I think the good one is simply : générer. Because all of the previous purpose seems to not be in the right context.

Reply:
@whispyy hi ! A retranslation is not needed as the issue was only with a grammatical error : the word "regénérer" does not exist, the correct word is "régénérer".

The text was actually given by the Content team, but I do like your version more, I'll use it !